src/components/chart/_macro.spec.js

```diff
@@ -1648,6 +1663,19 @@ describe('Macro: Chart', () => {
                     expect($('[data-invalid-chart-type]').length).toBe(0);
                 });
             });
+
+            describe('GIVEN: Params: Fallback Image URL with an iframe chart', () => {
```

Comment:
```suggestion
            describe('GIVEN: Params: fallbackImageUrl', () => {
```
This should be this, and then the a WHEN should mention the use with an iframe chart